q-files/vendors/govready/govready-q-files-startpack/q-files/System-Description-Demo/system_basics_technical.yaml

```diff
@@ -42,6 +42,28 @@ questions:
     - condition: application_where_hosted != 'cloud'
       value: ~
 
+  - id: aws_services
+    title: AWS Services
+    prompt: |
+      What AWS Services are being used?
+    type: multiple-choice
+    choices:
+      - key: EC2
+        text: Amazon EC2
+      - key: Lambda
+        text: AWS Lambda 
+      - key: RDS
+        text: AWS RDS
+      - key: EKS
+        text: Amazon Elastic Kubernetes Service (EKS)
```

Comment:
To match the other elements, this should be `Amazon Elastic Kubernetes Service` instead of `Amazon Elastic Kubernetes Service (EKS)`.

Reply:
I have pushed a new change with the suggested change. We can merge after.